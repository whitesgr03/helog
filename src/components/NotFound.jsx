import style from "../styles/Error.module.css";
import image from "../styles/utils/image.module.css";

const NotFound = () => {
	const darkTheme = false;
	return (
		<div className={`${darkTheme ? style.dark : ""} ${style.error}`}>
			<span className={`${image.icon} ${style.alert}`} />
			<h1 className={style.title}>404 Not Found</h1>
			<div className={style.message}>
				<p>Our apologies, there has been an error.</p>
				<p>The page you are looking for cannot be found.</p>
			</div>
		</div>
	);
};

export default NotFound;
