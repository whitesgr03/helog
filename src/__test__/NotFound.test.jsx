import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";

import NotFound from "../components/NotFound";

describe("NotFound component", () => {
	it("should render correctly", () => {
		const { asFragment } = render(<NotFound />);

		const actual = asFragment();

		expect(actual).toMatchSnapshot();
	});
});
