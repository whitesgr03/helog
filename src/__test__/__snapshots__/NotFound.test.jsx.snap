// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error component > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="_error_2fc9e1"
  >
    <span
      class="_icon_7e06e0 _alert_2fc9e1"
    />
    <h2>
      Page Not Found
    </h2>
    <div
      class="_message_2fc9e1"
    >
      <p>
        Our apologies, there has been an error.
      </p>
      <p>
        The page you are looking for could not be found.
      </p>
      <p>
        You may surf over to our other pages.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
